tests/data/snapshots/test_translate/admin/_language0_project0_translate_.context.json

```diff
@@ -0,0 +1,175 @@
+        },
+        {
+          "code": "tags_differ",
+          "title": "Tags differ"
+        },
+        {
+          "code": "template_format",
+          "title": "Template format"
+        },
+        {
+          "code": "test_check",
+          "title": "test_check"
+        },
+        {
+          "code": "unbalanced_curly_braces",
+          "title": "Curly braces"
+        },
+        {
+          "code": "unbalanced_tag_braces",
+          "title": "Unbalanced tag braces"
+        },
+        {
+          "code": "unescaped_ampersands",
+          "title": "Unescaped ampersands"
+        },
+        {
+          "code": "uppercase_placeholders",
+          "title": "Uppercase placeholders"
+        },
+        {
+          "code": "whitespace",
+          "title": "Whitespaces"
+        }
+      ],
+      "code": 100,
+      "name": "critical",
+      "title": "Critical"
+    },
+    {
+      "checks": [
+        {
+          "code": "doublequoting",
+          "title": "Double quotes"
+        }
+      ],
+      "code": 30,
+      "name": "cosmetic",
+      "title": "Cosmetic"
+    }
+  ],
+  "ctx_path": "/language0/project0/",
+  "editor_extends": "translation_projects/base.html",
+  "export_url": "/language0/project0/export-view/",
+  "has_admin_access": false,
+  "is_disabled": false,
+  "language": "Language 0 - language0",
+  "nav": "translation_projects/_nav.html",
+  "object": "/language0/project0/",
+  "page": "translate",
+  "pootle_path": "/language0/project0/",
+  "previous_url": "/",
+  "project": "Project 0",
+  "resource_path": "",
+  "resource_path_parts": [],
+  "search_action": "/language0/project0/translate/",
+  "search_form": "<tr><th><label for=\"id_search\">Search:</label></th><td><input id=\"id_search\" name=\"search\" placeholder=\"Search\" size=\"15\" title=\"Search (Ctrl+Shift+S)&lt;br/&gt;Type and press Enter to search\" type=\"text\" /></td></tr>\n<tr><th><label for=\"id_soptions_0\">Soptions:</label></th><td><ul id=\"id_soptions\"><li><label for=\"id_soptions_0\"><input id=\"id_soptions_0\" name=\"soptions\" type=\"checkbox\" value=\"exact\" /> Exact Match</label></li></ul></td></tr>\n<tr><th><label for=\"id_sfields_0\">Sfields:</label></th><td><ul id=\"id_sfields\"><li><label for=\"id_sfields_0\"><input checked=\"checked\" id=\"id_sfields_0\" name=\"sfields\" type=\"checkbox\" value=\"source\" /> Source Text</label></li>\n<li><label for=\"id_sfields_1\"><input checked=\"checked\" id=\"id_sfields_1\" name=\"sfields\" type=\"checkbox\" value=\"target\" /> Target Text</label></li>\n<li><label for=\"id_sfields_2\"><input id=\"id_sfields_2\" name=\"sfields\" type=\"checkbox\" value=\"notes\" /> Comments</label></li>\n<li><label for=\"id_sfields_3\"><input id=\"id_sfields_3\" name=\"sfields\" type=\"checkbox\" value=\"locations\" /> Locations</label></li></ul></td></tr>",
+  "translate_url": "/language0/project0/translate/",
+  "translation_project": "/language0/project0/",
+  "user": "nobody"
```

Comment:
I think this indicates some bug at least in the test code, as this should be the `admin` user. Also note how not all the `can{review,suggest,translate}` values are `true`.